strings_lab.js

```diff
@@ -0,0 +1,164 @@
+// # String Exercises
```

Comment:
I see some instances of hard coding where code is not adoptable to differing input. Always keep in mind to make your code workable with differing inputs. Keep practicing string methods as being able to manipulate string comfortably is super important for javascript mastery. Good job trying to answering bonus questions!